### PR TITLE
Hot fix integration bugs in templates and styles

### PR DESCRIPTION
**Pull request by McSinyx**
Sorry for the pile of unrelated changes:

- Fix use of project objects (participants -> students, supervisor -> supervisors)
- Revert some CSS change to for paragraphs formatting, input fields (reduce corner)
- Format user profile pages

Please review and I'll update accordingly as soon as possible to have 0.0.4 looking decent.

### CODE REVIEW

**Review by Huy-Ngo — APPROVED** [review]
OK I think the problems are solved.

File: src/acanban/static/style.css [hunk 1]
```diff
@@ -52,68 +52,50 @@ body {
...
+button, select, textarea,
+input[type=text], input[type=password], input[type=email] {
     background-color: var(--color-bg);
     border: 0.1rem solid var(--color-fg);
     box-sizing: border-box;
     color: var(--color-fg);
     margin: 0.5rem 0;
     padding: 0.5rem;
     width: 100%;
-    border-radius: 30px;
```
Comment (Huy-Ngo) [inline src/acanban/static/style.css:113]: Change border-radius to be consistent with the others (i.e. the search bar).

**Review by Huy-Ngo — CHANGES_REQUESTED** [review]

File: src/acanban/static/style.css (hunk 1 above)
Comment (McSinyx) [reply src/acanban/static/style.css:113]: Addressed.

**Review by Huy-Ngo — APPROVED** [review]

**Comment by 9a24f0** [comment]
LGTM!

**Review by 9a24f0 — APPROVED** [review]

**Pull request merged**